src/components/JobListing.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function JobListing({ listing }) {
+  return (
+    <div key={listing.id}>
+    <h1>{listing.title}</h1>
```

Comment:
I think we should make the position the first line of the listing and put the company name on the second line